--- rubber-1.1+20100306.orig/src/latex_modules/bibtex.py	2010-08-12 09:46:10.000000000 -0400
+++ rubber-1.1+20100306/src/latex_modules/bibtex.py	2012-08-07 19:06:01.000000000 -0400
@@ -407,7 +407,7 @@
 
 def setup (doc, context):
 	global biblio
-	biblio = Bibliography(doc, doc.target)
+	biblio = Bibliography(doc, basename(doc.target))
 	doc.hook_macro('bibliography', 'a', biblio.hook_bibliography)
 	doc.hook_macro('bibliographystyle', 'a', biblio.hook_bibliographystyle)
 def command (command, args):
