--- rubber-1.1+20100306.orig/src/latex_modules/index.py	2010-08-12 09:46:10.000000000 -0400
+++ rubber-1.1+20100306/latex_modules/index.py	2012-08-08 11:38:57.000000000 -0400
@@ -54,9 +54,9 @@
 		(e.g. .ilg) file.  Transcript is used by glosstex.py.
 		"""
 		self.doc = doc
-		self.source = doc.target + "." + source
-		self.target = doc.target + "." + target
-		self.transcript = doc.target + "." + transcript
+		self.source = os.path.basename(doc.target) + "." + source
+		self.target = os.path.basename(doc.target) + "." + target
+		self.transcript = os.path.basename(doc.target) + "." + transcript
 		if os.path.exists(self.source):
 			self.md5 = md5_file(self.source)
 		else:
